recipe/meta.yaml

```
@@ -21,9 +21,9 @@ build:
   skip: True
   {% endif %}
   script:
-    - cp {{ os.environ["FEEDSTOCK_ROOT"] }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt            # [linux]
-    - cp ${RECIPE_DIR}/../LICENSE.txt ${RECIPE_DIR}/LICENSE.txt                              # [osx]
-    - COPY "%RECIPE_DIR%\\..\\LICENSE.txt" "%RECIPE_DIR%\\LICENSE.txt"                       # [win]
+    - cp {{ os.environ.get("FEEDSTOCK_ROOT", "failme") }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt  # [linux]
```

We can just guard the usage if needed

```suggestion
    {% if "FEEDSTOCK_ROOT" in os.environ %}
    - cp {{ os.environ["FEEDSTOCK_ROOT"] }}/LICENSE.txt ${RECIPE_DIR}/LICENSE.txt  # [linux]
    {% endif %}
```

---

Let's move this comment to PR #350

---

Only responded here as we were having re-rendering issues: https://github.com/conda-forge/conda-forge-ci-setup-feedstock/pull/351#issuecomment-2375527805

---

yeah that is a new one that likely has the same underlying cause but is for a different variable...